hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -48,11 +48,13 @@ def __init__(self) -> None:
                                .format(dmem_size))
 
         # We represent the contents of DMEM as a list of 32-bit words (unlike
-        # the RTL, which uses a 256-bit array). An entry in self.data is None
-        # if the word has invalid integrity bits and we'll get an error if we
-        # try to read it. Otherwise, we store the integer value.
+        # the RTL, which uses a 256-bit array). An entry in self.data is a
+        # tuple of (value, validity). The value is an integer representing the
+        # 32-bit word, and the validity is a boolean indicating whether the
```

Comment:
Grammar nit: Here and on the next line, I think you can just say "..., and validity is ...". (Here, "validity" is the name of some variable. "The validity" means the correctness of something, which isn't quite what you want)